src/scene/shader-lib/chunks/gsplat/gsplatOutput.js

```diff
@@ -0,0 +1,42 @@
+export default /* glsl */`
+
+#if TONEMAP == FILMIC
+    #include "tonemappingFilmicPS"
+#elif TONEMAP == LINEAR
+    #include "tonemappingLinearPS"
+#elif TONEMAP == HEJL
+    #include "tonemappingHejlPS"
+#elif TONEMAP == ACES
+    #include "tonemappingAcesPS"
+#elif TONEMAP == ACES2
+    #include "tonemappingAces2PS"
+#elif TONEMAP == NEUTRAL
+    #include "tonemappingNeutralPS"
+#endif
+
+#if TONEMAP != NONE
+    #if GAMMA == SRGB
+        #include "decodePS"
+        #include "gamma2_2PS"
+    #else
+        #include "gamma1_0PS"
+    #endif
+#endif
+
+// prepare the output color for the given gamma-space color
+vec3 prepareOutputFromGamma(vec3 gammaColor) {
```

Comment:
How does this play with render passes? Don't render passes apply a tonemap as a post effect? Also, any reason not to apply fog?

Reply:
Currently only standard-material supports fog, but I could add this to gsplat shader.

Reply:
as long as the generator respects the related generation options (which it seems it is), it should work .. render passes pass the options they need